### PR TITLE
Read/unread for proxies

### PR DESCRIPTION
**Pull request by WilcoLouwerse**

### CODE REVIEW

**Review by smisidjan — APPROVED** [review]

File: Installation/Mapping/buren.pub.sourceOutQuery.mapping.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "openpub-type": "embedded&#46;acf&#46;publicationType.int_compare"
+  },
+  "unset": [
+    "_limit",
+    "_order",
+    "embedded&#46;acf&#46;publicationType",
+    "embedded&#46;acf&#46;publicationEndDate"
+  ],
+  "cast": {
+    "limit": "unsetIfValue==_limit",
```
Comment (rjzondervan) [inline Installation/Mapping/buren.pub.sourceOutQuery.mapping.json:20]: shouldn't this be per_page?

File: Installation/Mapping/buren.pub.typeSourceInBody.mapping.json [hunk 2]
```diff
@@ -2,10 +2,10 @@
   "title": "pub.typeSourceInBody",
   "$id": "https://buren.nl/mapping/buren.pub.typeSourceInBody.mapping.json",
   "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "passTrough": false,
   "mapping": {
-    "#": "[{% for object in # %}{% if not loop.first %},{% endif %}{{ map('https://buren.nl/mappings/openPubType.mapping.json', object)|json_encode }}{% endfor %}]"
+    "#": "[{% for key, object in _context %}{% if key matches '/^\\\\d+$/' %}{% if not loop.first %},{% endif %}{{ map('https://buren.nl/mapping/buren.pub.typeSourceInBodyItem.mapping.json', object)|json_encode }}{% endif %}{% endfor %}]"
```
Comment (rjzondervan) [inline Installation/Mapping/buren.pub.typeSourceInBody.mapping.json:8]: Fine for now, but we should come up with a better solution for _context mappings...

**Review by rjzondervan — COMMENTED** [review]
One question and one note for future reference.

File: Installation/Mapping/buren.pub.sourceOutQuery.mapping.json (hunk 1 above)
Comment (WilcoLouwerse) [reply Installation/Mapping/buren.pub.sourceOutQuery.mapping.json:20]: Yes you are right 👍  will fix

File: Installation/Mapping/buren.pub.typeSourceInBody.mapping.json (hunk 2 above)
Comment (WilcoLouwerse) [reply Installation/Mapping/buren.pub.typeSourceInBody.mapping.json:8]: Yeah, we should change it in a way that we always put everything we need for mapping in a _context.body parameter or something like that

Comment (rjzondervan) [reply Installation/Mapping/buren.pub.typeSourceInBody.mapping.json:8]: But keep backwards compatibility. I'm already a fan 😄

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**